.readthedocs.yaml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+
```

Comment:
@QuanMPhm according to the documentation linked it looks like we'll need to add a section for sphinx configuration. 

```foo.yaml
sphinx:
  configuration:
    docs/source/conf.py
```

Reply:
@naved001 I've made a test ReadTheDocs account of my own and got my fork to build successfully on their website.
My test documentation is live [here](https://documentation-quan-test.readthedocs.io/en/latest/networking/bu-ccds-networks.html)

Reply:
@QuanMPhm After some reading around, it looks like the reason your build works is because `sphinx` is the default documentation type, and if `sphinx.configuration` is not provided it looks for the conf file in the repo.
 
> If the value is null, Read the Docs will try to find a conf.py file in your project. 

https://docs.readthedocs.io/en/stable/config-file/v2.html#sphinx-configuration

If you look at the build for your test documentation, it can be seen that it shows the contents of the sphinx conf file at `docs/source/conf.py`. 

I want to be clear that sphinx's conf file is certainly being read and important for the build. @larsks do you have any opinions about if we should be explicit in referencing the sphinx config file or is it okay to rely on read the docs being smart enough to find it?

Reply:
@naved001 I mean...we've already been bitten once by something that used to be optional become required, so I guess that explicit is better than implicit. That said, since this pull request demonstrably fixes the problem, I would be happy to punt that to some future change.